controllers/articles-controllers.js

```diff
@@ -0,0 +1,8 @@
+const { fetchArticleById } = require("../models/articles-models.js")
+
+module.exports.getArticleById = (req, res, next) => {
+    const { article_id } = req.params;
+    return fetchArticleById(article_id).then((article) => {
+        res.status(200).send({article: article})
```

Comment:
this can be sent back as just {article} if you want

Reply:
This has been refactored.